### ISSUE TITLE
Add example visualization

### ISSUE THREAD

**Issue by andb0t** [umutphp/visualize-gitlabci#1]
Hi there,

I was looking for a tool similar to this and was wondering if you could (as part of your CI of course) make the tool run on your own repo and publish the visualization to github pages with a link to it in the `README.md` . Then everyone could see the current status.

Cheers!

**Comment by umutphp** [comment]
Hi @andb0t ,

Many thanks for your proposal. I will add the publishing task to the CI of the repo and update you asap.

**Comment by umutphp** [comment]
Hi @andb0t ,

I have checked how to do your request. It is a bit tricky because originally I am maintaining this package on [Gitlab.Com](http://gitlab.com). This repo is a mirror repo from the original on [Gitlab.Com](http://gitlab.com). But in short term, I have added the result of the execution of the comment for this repository and two other repositories as sample to the ReadMe file. 

I will try to make it dynamic as in your request and update you.
